planning/behavior_velocity_planner/config/speed_bump.param.yaml

```diff
@@ -0,0 +1,6 @@
+/**:
+  ros__parameters:
+    speed_bump:
+      slow_margin: 2.0
+      slow_velocity: 2.76 # 2.76 m/s = 10.0 kmph
```

Comment:
@yukkysaito cc : @beyzanurkaya 
We can set speed limit by map and suitable slowdown velocity is according to speed limit in area and speed bump height. And to insert speed limit in map would be more flexible and reasonable than applying constant slowdown velocity at speed bump area what do you think about this?

Reply:
@taikitanaka3 The plan is what you described here. There wont be a speed limit taken from yaml file. An attribute will be defined as speed_bump_height in the map under related speed_bump relation. So proper speed for the vehicle will be calculated in the module.

Reply:
@beyzanurkaya 
good 
> defined as speed_bump_height in the map under related speed_bump relation
I think we have at least 2 approaches.
1. set speed limit in map.
2. make slowdown behavior
eg.) set speed limit of 10km/h, 20km/h around speed bump area(actually it doesn't has to be speed bump area)
![image](https://user-images.githubusercontent.com/65527974/167521598-53a152e3-4d1d-40b2-a0b6-f4c075518706.png)

If the slow down speed is according to speed bump height we can set slow speed limit instead.(just comment)

Reply:
Hi @taikitanaka3,

I think setting speed limit from map is not proper for this purpose. Actually to accomplish that you don't even need to have a speed_bump module at all. However the purpose is to annotate speed_bump on lanelet2 map as a polygon (like a crosswalk) and have a module to handle it which calculates the proper speed wrt speed_bump height and makes sure that inside that polygon vehicle follows that speed. With this way if any kind of improvements or changes are needed it can be done in the module itself.

So the second option will be followed in this PR. But I didn't quite understand what do you mean with this:

> If the slow down speed is according to speed bump height we can set slow speed limit instead.(just comment)

Reply:
@mehmetdogru 
> If the slow down speed is according to speed bump height we can set slow speed limit instead
this comment was my first suggestion, sorry for confusion. I edited.

> Actually to accomplish that you don't even need to have a speed_bump module at all
annotation free is a good point.

> With this way if any kind of improvements or changes are needed
I am wondering the improvements that set slowdown velocity can't accomplish.
eg. ) 
- stop line need state machine to follow traffic rule
- no stopping area need predicted object info in order not to stop inside area.

Reply:
I'm not against to add this slow down module, I mean if there are some usecases in speed bump scene which set velocity limit in map is not a good idea, it's good to add it.（sorry for the ambiguity）.